test/system/010-images.bats

```diff
@@ -240,4 +240,33 @@ Labels.created_at | 20[0-9-]\\\+T[0-9:]\\\+Z
 
     run_podman rmi test:1.0
 }
+
+@test "podman images - rmi -af removes all containers and pods" {
+    # Make sure that force-removal of images removes associated containers and
+    # pods (in case of infra containers).
+    pname=$(random_string)
+    run_podman create --pod new:$pname $IMAGE
+
+    run_podman inspect --format '{{.ID}}' $IMAGE
+    imageID=$output
+
+    run_podman version --format "{{.Server.Version}}-{{.Server.Built}}"
+    pauseImage=localhost/podman-pause:$output
+    run_podman inspect --format '{{.ID}}' $pauseImage
+    pauseID=$output
+
+    run_podman rmi -af
```

Comment:
Could you attempt to `run_podman rmi -a` and make sure it fails with error about the Pod, then use the force.

Reply:
OMG speaking of The Force, the paranoid-test-mindset is strong in this one. Great thinking, @rhatdan